### PR TITLE
fixed zh-TW,Chinese (Taiwan) wording

### PR DESCRIPTION
**Pull request by DotnetChen**

### CODE REVIEW

File: locales/zh-TW.json [hunk 1]
```diff
@@ -4,18 +4,18 @@
   },
   "welcome": {
     "salutation": "你好，歡迎使用 Nook 手機上的 <b>Turnip Prophet</b>。",
-    "description": "這個程式可以讓你每天追蹤你島上的大頭菜價格，但你必須自己記錄價格！",
+    "description": "這個工具可以讓你每天追蹤自己島上的大頭菜價格，但你必須自己輸入價格！",
```
Comment (peter50216) [inline locales/zh-TW.json:7]: Should we also change all "你" to "您" for consistency with other part of the translations?

Comment (DotnetChen) [reply locales/zh-TW.json:7]: 這樣看起來統一成 "你" 可能會好一點? 您有什麼想法嗎?

Comment (peter50216) [reply locales/zh-TW.json:7]: 這邊我沒意見，我覺得只要統一一種寫法，"你"或"您"在這邊影響不大

Comment (DotnetChen) [reply locales/zh-TW.json:7]: 好的，那我統一改成 "你"

File: locales/zh-TW.json [hunk 2]
```diff
@@ -69,7 +69,7 @@
     }
   },
   "textbox": {
-    "description": "在您紀錄了一些大頭菜價格後，大頭菜預測工具會預測，並顯示你島上可能出現的不同模型。",
+    "description": "在你紀錄了一些大頭菜價格後，大頭菜預測工具會預測，並顯示自己島上可能出現的不同模型。",
```
Comment (peter50216) [inline locales/zh-TW.json:72]: Please also help change the "大頭菜預測工具" to "Turnip Prophet", thanks! (As in #167, we want the app name to be left untranslated.)

Comment (DotnetChen) [reply locales/zh-TW.json]: ok!

**Pull request merged**